proto/content_analysis/sdk/analysis.proto

```diff
@@ -74,10 +67,19 @@ message ContentAnalysisRequest {
   // The tags configured for the URL that triggered the content analysis.
   repeated string tags = 11;
 
+  // The data to analyze in the request. These fields are mutually exclusive,
+  // only one will be set in any content analysis request.
+  message ContentData {
+    // The text content to analyze.
+    optional string text_content = 1;
+    // The full path to the file to analyze. The path is expressed in a platform dependent way. 
+    optional string file_path = 2;
+  }
```

Comment:
I was thinking we would remove the ContentData message and just put the two fields directly into the request.  Is there a reason why we'd want to keep the message?

Another option might be to make this change to connectors.proto in the chromium repo first, and bringing the changes here.  That way we can get a review by chromium owners first.